jobs/src/main/scala/dcos/metronome/scheduler/SchedulerModule.scala

```diff
@@ -50,8 +50,7 @@ class SchedulerModule(
   private[this] lazy val shutdownHooks = ShutdownHooks()
   private[this] lazy val actorsModule = new ActorsModule(shutdownHooks, actorSystem)
 
-  private[this] lazy val eventModule: EventModule = new EventModule(scallopConf)
-  private[this] lazy val eventBus: EventStream = eventModule.provideEventBus(actorSystem)
+  private[this] lazy val eventBus: EventStream = actorSystem.eventStream
```

Comment:
we removed eventModule completely... marathon made it hard to use from marathon.. however the underlying impl was to get the eventStream.

Reply:
Seems better